.setup/distro_setup/setup_distro.sh

```diff
@@ -39,7 +39,7 @@ fi
 # check if set, else use default /etc/motd
 # we expect that SUBMISSION_URL and GIT_URL to be set above
 
-if [ ${VAGRANT} == 1 ]; then
+if [ ${VAGRANT} == 1] && [ ${UTM_ARM} == 0]; then
```

Comment:
For my curiosity, what was the error you hit here? I'm curious if UTM has a weird build of Ubuntu MOTD stuff, or what might be going on here.

Reply:
I'll get back to you when I retest this.  I might be because it wasn't inside of vagrant?  Or because I was re-running something.

I installed Ubuntu 20.04 Arm Server
https://mac.getutm.app/gallery/ubuntu-20-04